dev_environment/README.md

```diff
@@ -3,6 +3,8 @@
 The L3AF dev environment is a [Vagrant](https://www.vagrantup.com/) virtual
 machine environment that allows users to develop, test, or just try out L3AF.
 
+The Vagrant virtual machine environment only works for non Apple M1/M2 CPU based MAC computers.  A separate [shell script](https://github.com/l3af-project/l3af-arch/blob/main/dev_environment/Setup_Dev_Env.md) can set up the Linux virtual machine L3AF development environment for MAC/PC with Arm or x86 CPUs, Windows WSL and cloud based Linux VM (i.e. Azure Linux VM). 
+
```

Comment:
setup_dev_env_linux_vm.md is not referenced in this README.md and should be.  
I think this PR would obsolete most of PR #42, correct?  That PR does update README.md.

Reply:
Merged the PR#42 section of running rate limiting on the Azure VM.